tests/product/test_server_install.py

```diff
@@ -407,111 +389,6 @@ def test_install_interactive_with_hostnames(self):
             self.assert_has_default_connector(container)
             self.assert_has_jmx_connector(container)
 
-    def test_install_interactive_with_ips(self):
```

Comment:
it also tests the validation (which is also unit tested, so maybe the other test install with ips isn't necessary), but it's not _just_ a test of fabric